src/irep2/irep2_expr.h

```diff
@@ -2772,6 +2816,9 @@ class with2t : public with_expr_methods
     const expr2tc &value)
     : with_expr_methods(type, with_id, source, field, value)
   {
+#if 0 && !defined(NDEBUG)
```

Comment:
Remove 0.

Reply:
There still break lots of things with this enabled, also the check is too rigorous in its current form (e.g., update with an array having a size determined by a symbol). I'll remove this one and the other disabled check at `constant_struct2t` from this PR and make a new PR collecting the necessary changes regarding irep2-asserts.